### PR TITLE
Hello World in Julia

### PR DESCRIPTION
**Pull request by bipinkumar919**

### CODE REVIEW

File: VBScript.vbs [hunk 1]
```diff
@@ -0,0 +1 @@
+MsgBox "Hello, World!"
```
Comment (TiagoDanin) [inline VBScript.vbs]: Does not display `Hello World`, See [contributing guide](https://github.com/knightking100/hello-worlds/blob/master/CONTRIBUTING.MD)

**Comment by TiagoDanin** [comment]
Necessary separates in two PR (Julian and VBScript)

**Comment by TiagoDanin** [comment]
Reference issue #1

**Comment by bipinkumar919** [comment]
New to github.

File: Julia.jl [hunk 2]
```diff
@@ -0,0 +1 @@
+println("Hello world")
```
Comment (TiagoDanin) [inline Julia.jl]: `W` is uppercase

**Pull request merged**